packages/react/src/themes/teams/components/SplitButton/splitButtonVariables.ts

```diff
@@ -1,29 +1,30 @@
 import { SiteVariablesPrepared } from '../../../types'
-
 import { pxToRem } from '../../../../lib'
 
 export interface SplitButtonVariables {
   borderRadius: string
-  backgroundColorFocus: string
-  boxShadow: string
+  borderColorPrimary: string
   borderColor: string
-  colorFocus: string
-  primaryBackgroundColorFocus: string
-  primaryColorFocus: string
+  borderColorDisabled: string
+  borderWidthLeftOnly: string
+  smallDimension: string
+  smallPadding: string
+  smallMinWidth: string
+  smallBoxShadow: string
   padding: string
-  iconMargin: string
 }
 
 export default (siteVars: SiteVariablesPrepared): SplitButtonVariables => {
   return {
     borderRadius: siteVars.borderRadius,
-    backgroundColorFocus: siteVars.colors.grey[200],
-    boxShadow: siteVars.shadowLevel1,
-    borderColor: siteVars.colors.grey[200],
-    colorFocus: siteVars.colors.grey[750],
-    primaryBackgroundColorFocus: siteVars.colors.brand[800],
-    primaryColorFocus: siteVars.colors.white,
+    borderColor: siteVars.colorScheme.default.border,
+    borderColorPrimary: siteVars.colors.brand[500],
```

Comment:
Should this be a scheme value to change in different themes?

Reply:
It's ok as-is but also brand[500] doesn't match any colorscheme...